superduperdb/core/model.py

```
@@ -66,21 +72,32 @@ def __init__(
         except AttributeError:
             pass
 
-        if not hasattr(self, 'predict'):
+        if not hasattr(self, '_predict'):
             try:
-                self.predict = object.predict
+                self._predict = object._predict
             except AttributeError:
                 pass
```

So we just silently fail to do this, and hope everything after this point works?